ci-operator/config/stackrox/stackrox/stackrox-stackrox-nightlies__ocp-4-14.yaml

```diff
@@ -100,6 +101,17 @@ tests:
     - ref: stackrox-e2e
     workflow: ipi-aws
   timeout: 5h0m0s
+- as: sensor-profiling
+  postsubmit: true
+  steps:
+    cluster_profile: aws
+    env:
+      COLLECTION_METHOD: core_bpf
+      TEST_SUITE: ocp-sensor-profiling
+    test:
+    - ref: stackrox-e2e
+    workflow: ipi-aws
+  timeout: 2h0m0s
```

Comment:
This job is still a WIP as far as I know: https://prow.ci.openshift.org/job-history/gs/origin-ci-test/pr-logs/directory/pull-ci-stackrox-stackrox-master-ocp-4-13-sensor-profiling, so should not get added to nightlies until it has some passing history.